### ISSUE TITLE
Unable to replicate the results while training egogesture dataset

### ISSUE THREAD

**Issue by prabinaneupane** [ahmetgunduz/Real-time-GesRec#87]
Hello @ahmetgunduz, @parkjh688, @okankop 

Thank you for the amazing paper and the elaborate code. I am trying to replicate the results of your paper on the egogesture dataset. However, I am unable to get the accuracy even near to what you have achieved. Could you please help me figure out what I am doing wrong.

The parameters that I have used for training the classifier in the run-egogesture.sh are as follows:

python3 main.py --root_path ~/dataset/Real-time-GesRec-master \
    --video_path datasets/egoges/images \
	--annotation_path annotation_EgoGesture/egogestureall_but_None.json \
	--result_path results_final/ego \
	--pretrain_path pretrained_models/jester_resnext_101_RGB_32.pth \
	--dataset egogesture \
	--n_classes 27 \
	--n_finetune_classes 83 \
	--model resnext \
	--width_mult 1.0 \
	--model_depth 101 \
	--resnet_shortcut B \
	--resnext_cardinality 32 \
	--train_crop random \
	--learning_rate 0.01 \
	--sample_duration 32 \
	--modality RGB \
	--pretrain_modality RGB \
	--downsample 1 \
	--batch_size 24 \
	--n_threads 16 \
	--checkpoint 1 \
	--n_val_samples 1 \
    --n_epochs 30 \
    --ft_portion complete \
    --test \
    --lr_steps 10 25 \
    --test_subset test \
    --train_validate \

However the best precision and recall values for 30th epochs are as follows:

![image](https://user-images.githubusercontent.com/62377115/94112207-4cb55a80-fe6f-11ea-81e8-3e22116ef4c4.png)
